style.css

```diff
@@ -0,0 +1,315 @@
+body{
+    background-image: url('women.jpg');
+    background-repeat: no-repeat; /* Set background image to not repeat */
+    background-size: cover; /* Scale the background image to cover the entire viewport */
+    font-family: Arial, sans-serif;
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+
+*{
+    
+    margin: 0px;
+    padding: 0px;
+    box-sizing: border-box;
+    
+}  
+
+
+
+
+
+nav{
+    width: 100%;
+   /* height: 75px; */
+    /*line-height: 75px;*/
+    height: auto;
+    padding:10px 0;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    flex-wrap: wrap;
+
+    line-height:  2.6875rem;;
+    position: fixed;
+    background-image: linear-gradient(#033747,#012733);
+    border-bottom: 1px solid black;
+    box-shadow:0 0 10px rgba(250, 247, 247, 0.963); 
+    cursor: pointer;
+    
+}
+
+nav .logo p{
+    font-size: 30px;
+    font-weight: bold;
+    float: left;
+    color:white;

```

Comment:
using color codes is better for the future and get used to them from the beginning